docs/core/platform/analytics.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Analytics"
+---
+
+Home Assistant allows users to share their usage data via the analytics integration. The aggregated data is available at https://analytics.home-assistant.io. It is used to influence Home Assistant development priorities and to convince manufacturers to add local control and privacy-focused features.
+
+When attempting to communicate with manufacturers, it can be helpful to be equipped with information about what kind of a user base your integration has.
+
+For a user's installation of your integration to count the following conditions must be met:
+1) The integration must be set up.
```

Comment:
Use [markdown enumeration](https://www.markdownguide.org/basic-syntax/#lists-1), as it will otherwise bundle everything together as can be seen in [the preview](https://deploy-preview-1931--developers-home-assistant.netlify.app/docs/core/platform/analytics?_highlight=analyt).